tools/ci_build/github/azure-pipelines/templates/rocm-dev.yml

```diff
@@ -0,0 +1,92 @@
+# This pipeline product onnxruntime-training rocm python package with --enable_rocm_profiling.
+# It's used by ROCm EP developer.
+
+parameters:
```

Comment:
Is it possible to add a parameter to original file to avoid duplication?

Reply:
Yes, putting them into one file is a better way.